src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -1598,6 +1598,62 @@ SpannerSet MusicXMLParserPass2::findIncompleteSpannersAtPartEnd()
     return res;
 }
 
+//---------------------------------------------------------
+//   addArticLaissezVibrer
+//---------------------------------------------------------
+
+static void addArticLaissezVibrer(const Note* const note)
+{
+    IF_ASSERT_FAILED(note) {
+        return;
+    }
+
+    auto chord = note->chord();
```

Comment:
I'm not a fan of "auto". I use it only for complicated types like iterators etc, but when the type is as simple as Chord*, better make it explicit